lab-khalid/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "lab-khalid",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
+    "build": "./node_modules/webpack/bin/webpack.js"
```

Comment:
"lint": "./node_modules/eslint/bin/eslint.js ./**/*.js"

forgot to add lint script